awx/main/models/projects.py

```diff
@@ -466,6 +466,13 @@ def _get_related_jobs(self):
             models.Q(ProjectUpdate___project=self)
         )
 
+    def delete(self, *args, **kwargs):
+        path_to_delete = self.get_project_path(check_if_exists=False)
```

Comment:
@AlanCoding maybe we should *not* do this for manual projects, only SCM-based ones.  If they're *already* not using source control, I could see a scenario where somebody writes playbooks by hand, deletes a project, and then gets surprised by their hand-crafted playbooks vanishing.

Reply:
oh yeah, that's a _great_ point